dhis-2/dhis-services/dhis-service-administration/src/main/java/org/hisp/dhis/resourcetable/table/DatePeriodResourceTable.java

```diff
@@ -48,11 +48,11 @@
  * @author Lars Helge Overland
  */
 public class DatePeriodResourceTable
-    extends ResourceTable<Period>
+    extends ResourceTable<Integer>
 {
     private final String tableType;
 
-    public DatePeriodResourceTable( List<Period> objects, String tableType )
+    public DatePeriodResourceTable( List<Integer> objects, String tableType )
```

Comment:
We could name the List<Integer> as `years`, instead of `objects`.

Reply:
I think we are going with `objects` because of the superclass?